lib/engine/game/g_1860.rb

```diff
@@ -450,6 +528,56 @@ def compute_distance_graph(corporation)
         @hex_distances[corporation] = h_distances
       end
 
+      # needed for custom_node_walk
+      def custom_node_select(node, paths, corporation: nil)
+        on = paths.map { |p| [p, 0] }.to_h
+
+        custom_node_walk(node, on: on, corporation: corporation) do |path|
+          on[path] = 1 if on[path]
+        end
+
+        val = on.keys.select { |p| on[p] == 1 }
```

Comment:
you don't need the val, just do the selcet statement, you also don't need to do 0, just do nil, then you can do select { on[p] }

Reply:
Yep, code left over from when I was debugging.